cmd/skopeo/sync.go

```diff
@@ -104,6 +105,7 @@ See skopeo-sync(1) for details.
 	flags.StringVarP(&opts.destination, "dest", "d", "", "DESTINATION transport type")
 	flags.BoolVar(&opts.scoped, "scoped", false, "Images at DESTINATION are prefix using the full source image path as scope")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Copy all images if SOURCE-IMAGE is a list")
+	flags.BoolVarP(&opts.keepGoing, "keep-going", "", false, "Do not abort the sync if any image ref sync failed")
```

Comment:
```suggestion
	flags.BoolVarP(&opts.keepGoing, "keep-going", "", false, "Do not abort the sync if any image sync failed")
```
the `ImageReference` type is a Go-level concept users don’t know (and don’t need to care) about.

Maybe “image copy” instead of “image sync”, to more clearly differentiate between “the sync” and “image sync”.

(And should that be “failed” or “fails”? I’m not a native speaker.)

---

Well, the existing output does say “ref %d/%d”… I guess I’d prefer for this new option description not to introduce another instance, and to keep the progress output as is in this PR (changing the progress output text is not what this PR is about; we can do that later separately).

(Or, any other reviewers: feel free to override me and merge as is.)

Reply:
Done. I've updated this PR to use "image copy" and also changed "failed" to "fails" (I'm not a native English speaker either, but I think fails sounds more correct).